libcxx/test/std/thread/thread.mutex/thread.lock/thread.lock.guard/mutex.verify.cpp

```diff
@@ -6,8 +6,6 @@
 //
```

Comment:
Maybe rename to `ctor.mutex.pass.cpp` and test the constructor as a whole instead?

Reply:
Done.